pdr_backend/pdr_dashboard/util/format.py

```diff
@@ -0,0 +1,108 @@
+from typing import Union
+
+from enforce_typing import enforce_types
+from numerize import numerize
+
+
+@enforce_types
+def format_metric(metric: Union[int, float], value_id: str) -> str:
+    match value_id:
```

Comment:
I would create a config dict mapping value id to the function name (without the format_ part).
If the metric key is not in the config dict, thow an exception.
Otherwise call format_{function name}.

It helps with clarity and coroborated with my other comments it would look smth like:
{
"costs_metric": "percentage"
...
}

func_name = "format_" + formatting[value_id]
func = <get the function>
return func(metric)